app/Http/Controllers/Api/ReviewsController.php

```diff
@@ -43,6 +43,8 @@ class ReviewsController extends ApiController
      */
     public function __construct(PostService $posts, ReportbackService $reportbacks)
     {
+        $this->middleware('api');
```

Comment:
@DFurnes that was suggested here by @sbsmith86 so the `Routes` file would be a bit cleaner.